CHANGELOG.md

```diff
@@ -0,0 +1,23 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## Types of changes
+`Added ` for new features.
+`Changed` for changes in existing functionality.
+`Deprecated` for soon-to-be removed features.
+`Removed` for now removed features.
+`Fixed` for any bug fixes.
+`Security` in case of vulnerabilities.
+
+## [0.1.0] - 2024-01-09
```

Comment:
I remember that we should start the initial version at 0.1.1 as secretflow is using 0.1.0 now.

Reply:
This is a template only to show how to fill the content.
And i'm gona complete the content when release is ready (docs is not ready).

Reply:
Get.